index.d.ts

```diff
@@ -1574,6 +1582,34 @@ declare namespace Eris {
     filename?: string;
     id: string | number;
   }
+  interface Poll {
+    allow_multiselect: boolean;
+    answers: PollAnswer[];
+    expiry: string | null;
+    layout_type: 1;
```

Comment:
This still needs to be enumerated, not as a constant value. A new constants entry needs to be created